apps/tailwind-components/components/input/Ref.vue

```diff
@@ -223,66 +223,87 @@ prepareModel();
 
 <template>
   <InputGroupContainer @focus="emit('focus')" @blur="emit('blur')">
-    <div
-      class="flex flex-wrap gap-2 mb-2"
-      v-if="isArray ? selection.length : selection"
-    >
-      <Button
-        v-for="label in isArray ? selection : [selection]"
-        icon="cross"
-        iconPosition="right"
-        type="filterWell"
-        size="tiny"
-        @click="deselect(label as string)"
+    <template v-if="initialCount > limit">
+      <div
+        class="flex flex-wrap gap-2 mb-2"
+        v-if="isArray ? selection.length : selection"
       >
-        {{ label }}
-      </Button>
-    </div>
-    <div class="flex flex-wrap gap-2 mb-2">
-      <ButtonText @click="toggleSearch" :aria-controls="`search-for-${id}`">
-        Search
-      </ButtonText>
-      <ButtonText @click="clearSelection"> Clear all </ButtonText>
-    </div>
-    <template v-if="showSearch && initialCount > limit">
-      <InputLabel :for="`search-for-${id}`" class="sr-only">
-        search in {{ columnName }}
-      </InputLabel>
-      <InputSearch
-        :id="`search-for-${id}`"
-        :modelValue="searchTerms"
-        @update:modelValue="updateSearch"
-        class="mb-2"
-        :placeholder="`Search in ${columnName}`"
-        :aria-hidden="!showSearch"
-      />
+        <Button
+          @click="clearSelection"
+          v-if="isArray && selection.length > 1"
+          type="filterWell"
+          size="tiny"
+          icon="cross"
+          iconPosition="right"
+          class="mr-2"
+          >Clear all</Button
+        >
+        <Button
+          v-for="label in isArray ? selection : [selection]"
+          icon="cross"
+          iconPosition="right"
+          type="filterWell"
+          size="tiny"
+          @click="deselect(label as string)"
+        >
+          {{ label }}
+        </Button>
+      </div>
+      <div v-if="initialCount > limit" class="flex flex-wrap gap-2 mb-2">
```

Comment:
Like the ontology input, there's also content shifting. Add tw class `items-center justify-start`

Reply:
idem